dev-util/bencode/bencode_tools-20110315.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="C and Python tools for manipulating bencoded data"
+DESCRIPTION="bencode-tools is a collection of tools for manipulating \
+bencoded data. See http://en.wikipedia.org/wiki/Bencode for information \
+on bencode format.
+
+This was formely named typevalidator. With the introduction of \
+libbencodetools the project is named bencode-tools."
+HOMEPAGE="http://zakalwe.fi/~shd/foss/bencode-tools/"
+COPYRIGHT="2002-2005 Bram Cohen and Ross Cohen"
+LICENSE="LICENSE.bencode"
```

Comment:
Call this "Bencode License". Also, are you sure that it isn't just 3-clause BSD? It looks like that at first glance to me.